src/main/java/seedu/duke/command/ViewCommand.java

```diff
@@ -47,12 +50,38 @@ private void viewModulePlan(SelectedModulesList moduleList) {
             viewList.append(sem.getSem()).append(System.lineSeparator());
             for (Module selectedModule: sem) {
                 int index = sem.indexOf(selectedModule) + 1;
+                String doneIcon;
+                if (selectedModule.getDone()) {
+                    doneIcon = "[✓]";
+                } else {
+                    doneIcon = "[✗]";
+                }
                 viewList.append(index).append(".")
                         .append(selectedModule.toString())
+                        .append(doneIcon)
```

Comment:
Maybe we can override `toString` method in SelectedModule class. e.g: 
`@Override`
`public String toString() {`
`        return doneIcon + super().toString`
`}`
In this way, we don't need to `append(doneIcon)` in the view (or other) methods

Reply:
Thanks for the suggestion, I will change it!